### DISCUSSION TITLE
Testing Pygbag Support before Releasing

### DISCUSSION THREAD

**Discussion by BetterBuiltFool** [pygame-web/pygbag#185] [Q&A] [answered]
I'm working on adding pygbag support to a package I've made, and I'd like to test it in localhost before releasing it, to ensure that my fixes work as expected. However, the "dependencies" metadata downloads the release version from PyPI, which does not have the fixes.
This leaves me in a bind; I need to test my library before releasing it, but I need to release it before I can test it.

Is there a way to specify, such as in the metadata header, to use a local wheel or a TestPyPI version of a dependency? If not, is there any other way to test a prerelease library for compatibility, or do I need to do a quiet release, test, and delete the release if there are issues?

Thank you very much.

**Comment by pmp-p** [comment, marked as answer]
to test local changes, remove the library from the PEP block of main.py , unpack its wheel next to the test main.py and relaunch pygbag. Later if you edit files you don't need to relaunch but if you add or remove files to the unpacked wheel you must.

**Reply by BetterBuiltFool** [reply]
Excellent, that works perfectly. Thanks!
